### PR TITLE
himalaya 0.4.0 (new formula)

### LINKED ISSUE #147: Homebrew Support
It would be easier to install and upgrade the Himalaya cli if there was a formula in homebrew.

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

closes soywod/himalaya#147

### CODE REVIEW

File: Formula/himalaya.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+class Himalaya < Formula
+  desc "CLI email client in Rust"
```
Comment (nandahkrishna) [inline Formula/himalaya.rb:2]:
```suggestion
  desc "CLI email client written in Rust"
```

File: Formula/himalaya.rb [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  license "BSD-3-Clause"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    # default config from https://github.com/soywod/himalaya#configuration
```
Comment (nandahkrishna) [inline Formula/himalaya.rb:15]:
```suggestion
```
Can probably remove this.

**Review by nandahkrishna — COMMENTED** [review]
Just some minor suggestions.

**Comment by nandahkrishna** [comment]
I realised the comment may actually be useful, so I've added it back. Squashed the commits while I was at it 👍🏼.

**Review by nandahkrishna — APPROVED** [review]

**Comment by chenrui333** [comment]
> I realised the comment may actually be useful

no worries, IMO, that is meta-code 😄

**Review by SMillerDev — APPROVED** [review]

**Pull request closed without merging**